.evergreen/config/functions.yml

```diff
@@ -4,6 +4,11 @@ functions:
     - command: git.get_project
       params:
         directory: "src"
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: |
+          git submodule update --init
```

Comment:
I've just confirmed that this command checkout the submodule on the version committed in the main project. To update the submodule, I go to the directory and checkout another commit, then commit the reference to the main project.